src/NuGet.Indexing/Handlers/LatestListedHandler.cs

```diff
@@ -105,5 +111,11 @@ private static bool GetListed(Document document)
             string listed = document.Get("Listed");
             return (listed == null) ? false : listed.Equals("true", StringComparison.InvariantCultureIgnoreCase);
         }
+
+        private static bool GetSemver2(Document document)
+        {
+            string semverLevel = document.Get("SemverLevel");
```

Comment:
IsSemVer2 is probably more correct. GetListed specifically refers to retrieving the field named Listed from the document. The GetSemVer2 isn't actually getting a boolean field but rather getting a test field and running a test to return a boolean value. As such, IsSemVer2 makes more sense, and i've changed it to match.